public/app/plugins/datasource/loki/components/AnnotationsQueryEditor.tsx

```diff
@@ -34,6 +36,7 @@ export const LokiAnnotationsQueryEditor = memo(function LokiAnnotationQueryEdito
         history={[]}
         ExtraFieldElement={
           <LokiOptionFields
+            queryDirection={queryWithRefId.direction ?? datasource.direction ?? 'BACKWARD'}
```

Comment:
I think queryDirection={query.direction ?? datasource.direction} should be sufficient as we are always setting direction in datasource and already defaulting to BACKWARD there.

Reply:
Nice catch.